Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -332,7 +338,7 @@ class SignalAttachment: NSObject {
         for utiType in pasteboardUTISet {
             if UTTypeConformsTo(utiType as CFString, kUTTypeText) {
                 hasTextUTIType = true
-            } else {
+            } else if mediaUTISet.contains(utiType) {
                 hasNonTextUTIType = true
```

Comment:
I thought about it but I thought this was slightly more readable and I keep coming back in here and doing logging.